### DISCUSSION TITLE
I have an issue with the "compiled" / deployed version of my app but not on the local hot reload

### DISCUSSION THREAD

**Discussion by Hideman85** [pmndrs/react-spring#1204] [Support] [answered]
My issue is concerning `FrameLoop` file with the function `willAdvance`

```js
TypeError: r.willAdvance is not a function
    update FrameLoop.ts:164
    gi React
    update FrameLoop.ts:160
    a FrameLoop.ts:248
```

I use it in combination with drag & drop event to provide a swipe action between containers horizontally aligned.

There is an extract of my code:
```js
const [props, set] = useSprings(children.length, () => ({
    x: 0,
    scale: 1,
    display: 'block'
  }))

const tickUpdate = () => {
  set(i => {
    if (i < index.current - 1 || i > index.current + 1) {
      return { display: 'none' }
    }
    const x = (i - index.current) * root.current.clientWidth + innerProps.distance
    const scale = 1 - Math.abs(innerProps.distance) / root.current.clientWidth / 2
    return { x, scale, display: 'block' }
  })
```

My issue like title said happen only on the deployed version (not on the local hot reload version) and I'm using the latest webpack `@svgr/webpack^5.4.0`
Did you guys faced to the same issue? does webpack in a way lost the reference to the function? or simply eat code?

**Comment by aleclarson** [comment, marked as answer]
See here: https://github.com/pmndrs/react-spring/issues/1078
